cmd/bnscli/clitests/account_register_domain.test

```diff
@@ -5,7 +5,7 @@ set -e
 bnscli register-domain \
 		-domain "mydomain" \
 		-admin "92066456B2BE7F1934624087D98C203A87F7752C" \
-		-superuser \
+		-superuser "false" \
```

Comment:
Seems to me that this should have been "true" and the following superuser should be "false".